tests/lib/SimpleSAML/ConfigurationTest.php

```diff
@@ -31,6 +31,13 @@ public function testLoadDefaultInstance()
      */
     public function testCriticalConfigurationError()
     {
+        try {
+            SimpleSAML_Configuration::getInstance();
+            $this->fail('Exception expected');
+        } catch (\SimpleSAML\Error\CriticalConfigurationError $var) {
+            // This exception is expected.
```

Comment:
Yup. This part of the code change was related to reducing state spill over.  This test previously was relying on testLoadDefaultInstance() running before it so it would in the correct state for this test. Now the test can be run on its own.